dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNBusInterfaceIIDM.h

```diff
@@ -121,6 +121,13 @@ class BusInterfaceIIDM : public BusInterface {
     return empty;
   }
 
+  /**
+   * @copydoc BusInterface::isFictitious() const
+   */
+  bool isFictitious() const {
```

Comment:
styling: In c++11 part, overriden functions should be marked as "final"

Reply:
Added `final` specifier to the PowSyBl IIDM Bus classes that override `isFictitious` in dynawo data interface.